scripts/irods/test/test_resource_types.py

```diff
@@ -2385,6 +2385,142 @@ def test_auto_repl_on__2941(self):
         self.admin.assert_icommand("ils -L " + filename, 'STDOUT_SINGLELINE', 'cacheResc')
         self.admin.assert_icommand("ils -L " + filename, 'STDOUT_SINGLELINE', 'archiveResc')
 
+    def test_get_stale_cache_replica__issue_6502(self):
+
+        try:
+            filename = "testgetstalecachereplica.txt"
+            lib.make_file(filename, 10, 'arbitrary')
+            stale_replica_filename = "stalereplica.txt"
+            lib.make_file(stale_replica_filename, 20, 'arbitrary')
```

Comment:
Should these calls to `lib.make_file` be replaced with `lib.make_arbitrary_file`? Otherwise, should we remove `lib.make_arbitrary_file` since it's not being used?

Reply:
The make_arbitrary_file was newly created.  It was initially created in the S3 resource to provide better files for testing.  I think in the long run the S3 will just point to the this new version in lib.

I'm not sure if these tests require make_arbitrary_file but some in the future may.